cocotb/handle.py

```diff
@@ -577,6 +577,26 @@ def loads(self):
         """An iterator for gathering all loads on a signal."""
         return _SimIterator(self._handle, simulator.LOADS)
 
+class SetAction:
+    """Base class representing the type of action used while setting a handle value."""
```

Comment:
Marginal performance increase, maybe:
```suggestion
    """Base class representing the type of action used while setting a handle value."""
    __slots__ = ("value",)
```